fission/main.go

```diff
@@ -41,11 +46,25 @@ func getKubeConfigPath() string {
 	return kubeConfig
 }
 
+func getFissionAPIVersion(apiUrl string) string {
+	resp, err := http.Get(apiUrl)
+	if err != nil {
+		fatal(fmt.Sprintf("Error connection Fission API server: %v", err))
+	}
+	defer resp.Body.Close()
+
+	body, err := ioutil.ReadAll(resp.Body)
+	if err != nil {
+		fatal(fmt.Sprintf("Error parsing Fission API version: %v", err))
```

Comment:
I don't think completely crashing is the best option here; the user might still want to know their client version. Currently it will just crash, not showing the client version either.

Just make this a multivalue return with an error, and handle error in the VersionPrinter.